### PR TITLE
EPMRPP-56172 Send logs on test fail

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: src/reporter.ts [hunk 1]
```diff
@@ -35,7 +36,7 @@ export class Reporter {
   private testItems: TestItem[];
 
   constructor(config: ReportPortalConfig) {
-    this.noColors = true;
+    this.noColors = false;
```
Comment (KatsiarynaTatarynovich) [inline src/reporter.ts:39]: Why did you change this value?

**Review by KatsiarynaTatarynovich — CHANGES_REQUESTED** [review]

File: src/reporter.ts (hunk 1 above)
Comment (AmsterGet) [reply src/reporter.ts:39]: It's just a marker that controls the console output, colored or not.
Here I decided that colored output will be more useful :)

**Review by KatsiarynaTatarynovich — APPROVED** [review]
